src/test/components/NetworkChart.test.js

```diff
@@ -680,3 +691,80 @@ describe('Network Chart/tooltip behavior', () => {
     expect(queryByTestId('tooltip')).toBeFalsy();
   });
 });
+
+describe('calltree/ProfileCallTreeView navigation keys', () => {
+  beforeEach(addRootOverlayElement);
+  afterEach(removeRootOverlayElement);
+
+  function setup(markers) {
+    const { container } = setupWithPayload(markers);
+
+    const renderedRows = container.querySelectorAll('.networkChartRowItem');
+    expect(renderedRows.length).toEqual(48);
+
+    return {
+      // take either a key as a string, or a full event if we need more
+      // information like modifier keys.
+      simulateKey: (param: string | { key: string }) => {
+        const treeViewBody = ensureExists(
+          container.querySelector('div.treeViewBody'),
+          `Couldn't find the tree view body with selector .networkChart`
+        );
+        fireEvent.keyDown(
+          treeViewBody,
+          // There's a shortcoming in either Flow or the flow type for the
+          // `keyDown` method. $FlowExpectError
+          param.key ? param : { key: param }
+        );
+      },
+      selectedText: () =>
+        ensureExists(
+          container.querySelector('.isSelected'),
+          `Couldn't find the selected row with selector .isSelected`
+        ).textContent,
+    };
+  }
```

Comment:
Can you please leave some empty lines before and after cases? This makes the file less dense and easier to read, as we see more immediately the chunks that make the cases.